corehq/apps/hqwebapp/management/commands/list_waf_allow_patterns.py

```diff
@@ -21,7 +21,7 @@ def handle(self, *args, **options):
                 else:
                     # @waf_allow(kind)
                     for urlmatch in resolver.reverse_dict.getlist(view):
-                        patterns.append(resolver.regex.pattern + urlmatch[1])
+                        patterns.append(str(resolver.pattern) + urlmatch[1])
```

Comment:
@proteusvacuum any idea what changed here? Was it a Django upgrade?

Reply:
Yes, but apparently this was a change in Django 2.0. It's surprising that this hasn't been run since we upgraded...

https://github.com/getsentry/raven-python/issues/1127

Reply:
Spooky. I know it was run just recently. I wonder if people have been fixing it for themselves locally and assuming it was their fault somehow and not PRing it?